website/docs/docs/dbowser-demo.md

```diff
@@ -0,0 +1,85 @@
+
+- `hubspot__marketing_email_campaign`: The raw table of marketing emails and their connection to campaigns in HubSpot.
+- `hubspot__marketing_email`: The raw table of all marketing emails and their details (content, author, created date, etc.) in HubSpot, not necessarily connected to campaigns.
+- `hubspot__email_event:` A log table of all email events (open/sends/clicks, etc.) with a unique id per email event.
+- `hubspot__email_event_sent`: A log of all emails sent from your HubSpot account.
+- `hubspot__contact`: The list of contacts in your HubSpot account, as well as some relevant information, such as first name and last name.
+
+For the sake of this tutorial, we’re loading in our sources as [seeds](https://docs.getdbt.com/docs/build/seeds), static CSV files dbt can load into your data warehouse. In reality, if you were doing this with your own HubSpot (or other email platform) connector (via some Fivetran, Stitch, or other ETL tool), you would have this raw data already in your data warehouse and would treat them as [sources](https://docs.getdbt.com/docs/build/sources) in dbt.
+
+Use this first IDE to explore these raw data sources, perform some exploratory data analysis (EDA), and get familiar with your dbt project. No wrong answers here 🙂
+
+<dbtEditor project="dbt_sample_project" step="1" />
+
+## Step 2: Build your staging models
+
+As with any new modeling project in dbt, we always recommend by creating a 1-1 mapping of raw sources with new [staging models](https://docs.getdbt.com/guides/best-practices/how-we-structure/2-staging). These models are meant to perform relatively simple cleanup (type recasting, column renaming, rounding, etc.); they’re not meant to be performing any joins or complex logic.
+
+At this stage, your goal in these models is to simply ensure our raw email source tables have the correct columns selected (and named appropriately), consistent naming conventions are applied across models for similar columns, and filtering of deleted records removed. (We always recommend filtering for removed records earlier in your DAG to improve performance in downstream models).
+
+At the end of this modeling stage, you should have 5 new models (downstream from your sources):
+
+- `stg_hubspot__contacts.sql`
+- `stg_hubspot__marketing_email_campaigns.sql`
+- `stg_hubspot__marketing_emails.sql`
+- `stg_hubspot__email_events.sql`
+- `stg_hubspot__email_event_sent.sql`
+
+How would you go about ensuring columns are named properly across models? Which columns do you think you need selected to perform downstream joins?
+
+Use the IDE below to create the staging models listed above. Remember: simpler is better (when it comes to most things in SQL and definitely in staging models 😉).
+
+<dbtEditor project="dbt_sample_project" step="2" />
```

Comment:
Let's rename `step` to `tag` here (and read the full tag name, ie `step_1`) for universality

Reply:
Updated prop to `tag`. Think it's worth adding support for multiple tags within this prop?